test-data/genops-basic.test

```diff
@@ -588,16 +588,18 @@ def f(x):
     x :: int
     r0 :: object
     r1 :: str
-    r2, r3, r4 :: object
-    r5 :: int
+    r2 :: dict
+    r3, r4, r5 :: object
+    r6 :: int
```

Comment:
This doesn't sound like a very useful test, right? It looks like a glorified "golden file" check.

Reply:
The genops tests are useful in the following ways:
1. When introduced, you can check them to make sure that IR looks like you want---typically, that some optimized path is properly used.
2. It prevents the IR we generate for something from changing by accident.

It is also not great in a lot of ways. It is not always clear from a test what the salient features we care about in the IR are, and the test itself isn't discriminating about it. To make updating expected test outputs feasible, we have an automated system for updating them, and for high-churn tests it is easy to miss something.

I've opened #486 about finding a better way